myokit/formats/sbml/_importer.py

```diff
@@ -37,332 +38,1016 @@ def __init__(self):
         self.re_white = re.compile(r'[ \t\f\n\r]+')
         self.units = {}
 
-    def _convert_name(self, name):
-        """
-        Converts a name to something acceptable to myokit.
-        """
-        if not self.re_name.match(name):
-            org_name = name
-            name = self.re_white.sub('_', name)
-            name = self.re_alpha.sub('_', name)
-            if not self.re_name.match(name):
-                name = 'x_' + name
-            self.logger().warn(
-                'Converting name <' + org_name + '> to <' + name + '>.')
-        return name
-
-    def _convert_unit(self, unit):
-        """
-        Converts an SBML unit to a myokit one using the lookup tables generated
-        when parsing the XML file.
-        """
-        if unit in self.units:
-            return self.units[unit]
-        elif unit in unit_map:
-            return unit_map[unit]
-        else:      # pragma: no cover
-            raise SBMLError('Unit not recognized: ' + str(unit))
-
     def info(self):
         return info
 
-    def model(self, path, bind_time=True):
+    def model(self, path):
         """
         Returns a :class:myokit.Model based on the SBML file provided.
 
         Arguments:
-
-        ``path``
-            The path to the SBML file.
-        `` bind_time``
-            If set to ``True`` (default), a variable called "time" will be
-            created and bound to `time`.
-
+            path -- Path to SBML file.
```

Comment:
Please update to red code above, to match the style used in the rest of Myokit :-)